packages/medusa/src/api-v2/admin/products/query-config.ts

```diff
@@ -30,6 +30,10 @@ export const defaultAdminProductFields = [
   // TODO: Handle in next iteration
   // "collection_id",
   // "type_id",
+  // TODO: Can I expand the fields list from the HTTP request, or does it get replaced instead? What purpose do the allowed and default relations fields have?
```

Comment:
I don't want these in the defaults so we have 1:1 compatability with v1, I just added them so my test passes until I get an answer on how these relationships should be expanded with remoteQuery

Reply:
As I mentioned somewhere, in the future the idea will be that maybe a specific props on the req could allow a user to alter allowed and default. Also, from the http currently there is fields and expand, but the idea would be to only keep one and also have something (i proposed +,-, nothing as prefix of props and relation if needed) to alter the default fields and not just relpace all unless there is no prefix. Once we have that and only a single array instead of default relations and default fields which can be merged, then we can update the transformer to always provider both `select, relations` and `fields` which can be used with the remote query object utils to transform it and it will auto include the user request

Reply:
don't know if I explained it well but we can discuss that later (I can whisper only ahah)

Reply:
Yeah I understand, I'll just need a concrete example to see how all of this would look like, I'll get lunch now let's chat later 😄